lighthouse-cli/test/smokehouse/test-definitions/dobetterweb/dbw-expectations.js

```diff
@@ -434,7 +434,7 @@ const expectations = [
         mixedContent: [
           {
             resourceType: 'Image',
-            resolutionStatus: 'MixedContentWarning',
+            resolutionStatus: 'MixedContentAutomaticallyUpgraded',
```

Comment:
heh. fixed the ToT bots and broke the stable ones.. 

```suggestion
            _minChromiumMilestone: 88, # We went from Warning to AutoUpgrade in https://chromium-review.googlesource.com/c/chromium/src/+/2480817
            resolutionStatus: 'MixedContentAutomaticallyUpgraded',
```

i think this'll work?